src/uu/dircolors/src/dircolors.rs

```diff
@@ -135,13 +135,15 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let result;
     if files.is_empty() {
         result = parse(INTERNAL_DB.lines(), &out_format, "");
+    } else if files.len() > 1 {
+        return Err(UUsageError::new(
```

Comment:
is it possible to add a test to trigger this error ?
thanks

Reply:
of course